core/src/main/scala/org/apache/spark/status/api/v1/api.scala

```diff
@@ -111,7 +115,11 @@ class RDDDataDistribution private[spark](
     val address: String,
     val memoryUsed: Long,
     val memoryRemaining: Long,
-    val diskUsed: Long)
+    val diskUsed: Long,
+    val onHeapMemoryUsed: Long,
+    val offHeapMemoryUsed: Long,
+    val onHeapMemoryRemaining: Long,
+    val offHeapMemoryRemaining: Long)
```

Comment:
These fields still use `Long` instead of `Option`, since only Live UI could access such REST API, and in the live UI everything is up to date.